examples/js/loaders/GLTFLoader.js

```diff
@@ -2262,6 +2262,14 @@ THREE.GLTFLoader = ( function () {
 
 			if ( materialDef.name !== undefined ) material.name = materialDef.name;
 
+			// Normal map textures use OpenGL conventions:
+			// https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#materialnormaltexture
```

Comment:
I am not sure that comment explains the ensuing code block...

Maybe referencing https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995 would be OK?